Tests/StreamChatTests/Controllers/ChannelListController/ChannelListController_Tests.swift

```diff
@@ -1095,6 +1092,268 @@ final class ChannelListController_Tests: XCTestCase {
+        )
+    }
+
+    func test_filterPredicate_greaterOrEqual_containsExpectedItems() throws {
+        let cid1 = ChannelId.unique
+        let cid2 = ChannelId.unique
+
+        try assertFilterPredicate(
+            .greaterOrEqual(.memberCount, than: 1),
+            channelsInDB: [
+                .dummy(channel: .dummy(cid: cid1, members: [.dummy(), .dummy()])),
+                .dummy(channel: .dummy(cid: cid2, members: [.dummy()])),
+                .dummy(channel: .dummy(members: []))
+            ],
+            expectedResult: [cid1, cid2]
+        )
+    }
+
+    func test_filterPredicate_less_containsExpectedItems() throws {
+        let cid = ChannelId.unique
+
+        try assertFilterPredicate(
+            .less(.memberCount, than: 1),
+            channelsInDB: [
+                .dummy(channel: .dummy(members: [.dummy(), .dummy()])),
+                .dummy(channel: .dummy(members: [.dummy()])),
+                .dummy(channel: .dummy(cid: cid, members: []))
+            ],
+            expectedResult: [cid]
+        )
+    }
+
+    func test_filterPredicate_lessOrEqual_containsExpectedItems() throws {
+        let cid1 = ChannelId.unique
+        let cid2 = ChannelId.unique
+
+        try assertFilterPredicate(
+            .lessOrEqual(.memberCount, than: 1),
+            channelsInDB: [
+                .dummy(channel: .dummy(members: [.dummy(), .dummy()])),
+                .dummy(channel: .dummy(cid: cid1, members: [.dummy()])),
+                .dummy(channel: .dummy(cid: cid2, members: []))
+            ],
+            expectedResult: [cid1, cid2]
+        )
+    }
+
+    func test_filterPredicate_in_containsExpectedItems() throws {
+        let memberId1 = UserId.unique
+        let memberId2 = UserId.unique
+        let cid = ChannelId.unique
+
+        try assertFilterPredicate(
+            .in(.members, values: [memberId1, memberId2]),
+            channelsInDB: [
+                .dummy(channel: .dummy(members: [.dummy(), .dummy()])),
+                .dummy(channel: .dummy(members: [.dummy()])),
+                .dummy(channel: .dummy(cid: cid, members: [.dummy(user: .dummy(userId: memberId1)), .dummy(user: .dummy(userId: memberId2))]))
+            ],
+            expectedResult: [cid]
+        )
+    }
+
+    func test_filterPredicate_notIn_containsExpectedItems() throws {
+        let memberId1 = UserId.unique
+        let memberId2 = UserId.unique
+        let cid1 = ChannelId.unique
+        let cid2 = ChannelId.unique
+
+        try assertFilterPredicate(
+            .notIn(.members, values: [memberId1, memberId2]),
+            channelsInDB: [
+                .dummy(channel: .dummy(cid: cid1, members: [.dummy(), .dummy()])),
+                .dummy(channel: .dummy(cid: cid2, members: [.dummy()])),
+                .dummy(channel: .dummy(members: [.dummy(user: .dummy(userId: memberId1)), .dummy(user: .dummy(userId: memberId2))]))
+            ],
+            expectedResult: [cid1, cid2]
+        )
```

Comment:
is this working? Would be nice to member1 to cid1 and member2 to cid2

Reply:
Yeap this is working as expected

Reply:
I think having member1 in cid1 and member2 in cid2 can be a good test so we make sure it not treated as an "OR", but nothing else aside from that